lib/eex/test/eex_test.exs

```diff
@@ -81,123 +82,139 @@ defmodule EExTest do
-        EEx.compile_string "foo <% end %>"
+      assert_raise EEx.SyntaxError, "nofile:1: unexpected end of expression <% end %>", fn ->
+        EEx.compile_string("foo <% end %>")
       end
     end
 
     test "when start expression is found without an end expression" do
-      assert_raise EEx.SyntaxError, "nofile:2: unexpected end of string, expected a closing '<% end %>'", fn ->
-        EEx.compile_string "foo\n<% if true do %>"
-      end
+      assert_raise EEx.SyntaxError,
+                   "nofile:2: unexpected end of string, expected a closing '<% end %>'",
+                   fn ->
+                     EEx.compile_string("foo\n<% if true do %>")
+                   end
     end
 
     test "when nested end expression is found without a start expression" do
       assert_raise EEx.SyntaxError, "nofile:1: unexpected end of expression <% end %>", fn ->
-        EEx.compile_string "foo <% if true do %><% end %><% end %>"
+        EEx.compile_string("foo <% if true do %><% end %><% end %>")
       end
     end
 
     test "when middle expression has a modifier" do
       assert ExUnit.CaptureIO.capture_io(:stderr, fn ->
-        EEx.compile_string "foo <%= if true do %>true<%= else %>false<% end %>"
-      end) =~ ~s[unexpected beginning of EEx tag \"<%=\" on \"<%= else %>\"]
+               EEx.compile_string("foo <%= if true do %>true<%= else %>false<% end %>")
+             end) =~ ~s[unexpected beginning of EEx tag \"<%=\" on \"<%= else %>\"]
     end
 
     test "when end expression has a modifier" do
       assert ExUnit.CaptureIO.capture_io(:stderr, fn ->
-        EEx.compile_string "foo <%= if true do %>true<% else %>false<%= end %>"
-      end) =~ ~s[unexpected beginning of EEx tag \"<%=\" on end of expression \"<%= end %>\"]
+               EEx.compile_string("foo <%= if true do %>true<% else %>false<%= end %>")
+             end) =~
+               ~s[unexpected beginning of EEx tag \"<%=\" on end of expression \"<%= end %>\"]
     end
 
     test "when trying to use marker '/' without implementation" do
       assert_raise EEx.SyntaxError,
-        ~r/unsupported EEx syntax <%\/ %> \(the syntax is valid but not supported by the current EEx engine\)/, fn ->
-          EEx.compile_string "<%/ true %>"
-      end
+                   ~r/unsupported EEx syntax <%\/ %> \(the syntax is valid but not supported by the current EEx engine\)/,
+                   fn ->
+                     EEx.compile_string("<%/ true %>")
+                   end
     end
 
     test "when trying to use marker '|' without implementation" do
-      assert_raise EEx.SyntaxError, 
-        ~r/unsupported EEx syntax <%| %> \(the syntax is valid but not supported by the current EEx engine\)/, fn ->
-          EEx.compile_string "<%| true %>"
-      end
+      assert_raise EEx.SyntaxError,
+                   ~r/unsupported EEx syntax <%| %> \(the syntax is valid but not supported by the current EEx engine\)/,
```

Comment:
Can you please move the error message out? So it looks like this:

```
msg =  ~r/unsupported EEx syntax <%| %> \(the syntax is valid but not supported by the current EEx engine\)/
assert_raise EEx.SyntaxError, msg, fn -> EEx.compile_string("<%| true %>") end
```

This needs to be done in this line and the ones below.